x-pack/plugin/runtime-fields/src/main/java/org/elasticsearch/xpack/runtimefields/query/AbstractLongScriptFieldQuery.java

```diff
@@ -41,7 +41,7 @@
     /**
      * Does the value match this query?
      */
-    protected abstract boolean matches(long[] values, int count);
+    protected abstract boolean matches(long[] values);
```

Comment:
I feel like we'll want this back at some point, but it is easy enough to add back in.